src/pages/api/item/index.js

```diff
@@ -29,77 +29,67 @@ const addItem = async (req, res) => {
     }
 
-    browser = await launchChromium({
-      headless:true,
-      args: [
-        '--disable-gpu',
-        '--disable-dev-shm-usage',
-        '--disable-setuid-sandbox',
-        '--no-first-run',
-        '--no-sandbox',
-        '--no-zygote',
-        '--deterministic-fetch',
-        '--disable-features=IsolateOrigins',
-        '--disable-site-isolation-trials',
-        '--hide-scrollbars', 
-        '--disable-web-security',
-        "--window-size=2000x1500"
-    ],
-    });
+    const chromeOptions = new chrome.Options();
+    chromeOptions.addArguments([
+      '--disable-gpu',
+      '--no-sandbox',
+      '--disable-dev-shm-usage',
+      '--disable-setuid-sandbox',
+      '--no-first-run',
+      '--no-zygote',
+      '--deterministic-fetch',
+      '--disable-features=IsolateOrigins',
+      '--disable-site-isolation-trials',
+      '--hide-scrollbars', 
+      '--disable-web-security',
+      '--window-size=2000x1500',
+      '--ignore-certificate-errors',
+      '--headless',
+    ]);
+
+    driver = await new Builder()
+      .forBrowser('chrome')
+      .setChromeOptions(chromeOptions)
+      .build();
+
+    await driver.get(link);
 
-    //  browser = await Promise.resolve(bundledChromium.executablePath)
-    // .then((executablePath) =>
-    //   !executablePath  
-    //     ? chromium.launch({ headless:true,
-    //       args: [
-    //         '--disable-gpu',
-    //         '--disable-dev-shm-usage',
-    //         '--disable-setuid-sandbox',
-    //         '--no-first-run',
-    //         '--no-sandbox',
-    //         '--no-zygote',
-    //         '--deterministic-fetch',
-    //         '--disable-features=IsolateOrigins',
-    //         '--disable-site-isolation-trials',
-    //         '--hide-scrollbars', 
-    //         '--disable-web-security',
-    //         "--window-size=2000x1500"
-    //     ], })
-    //     : chromium.launch({ executablePath })
-    // );
-
-    context = await browser.newContext({
-      userAgent:
-        "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36",
-    });
-    page = await context.newPage();
-  
-      await page.goto(link, {waitUntil: "networkidle"});
+    try {
+      await driver.wait(until.titleContains('Expected Page Title'), 15000);
+    } catch (error) {
+      console.error("Error during waiting condition:", error);
+    }
 
-      let title, description, image;
+    let title = 'Title Not Found';
+    let description = 'Description Not Found';
+    let image = 'https://i.imgur.com/Ki1kaw4.png';
 
     try {
-      title = await page.$eval('meta[property="og:title"]', (element) => element.getAttribute('content'));
+      const titleElement = await driver.findElement(By.css('meta[property="og:title"], meta[name="title"]'));
+      title = await titleElement.getAttribute('content');
     } catch (error) {
-      title = "Title Not Found";
```

Comment:
Good change. In other places too. Nice catch to not set it when we already have default values check again